server/log_operation_manager.go

```diff
@@ -185,31 +185,38 @@ func (l *LogOperationManager) masterFor(ctx context.Context, allIDs []int64) ([]
 	if l.info.Registry.ElectionFactory == nil {
 		return allIDs, nil
 	}
+	allIDString := make([]string, 0, len(allIDs))
+	idToString := make(map[int64]string)
+	for _, id := range allIDs {
+		s := strconv.FormatInt(id, 10)
+		allIDString = append(allIDString, s)
+		idToString[id] = s
+	}
 	if l.tracker == nil {
 		glog.Infof("creating mastership tracker for %v", allIDs)
-		l.tracker = election.NewMasterTracker(allIDs, func(id int64, v bool) {
+		l.tracker = election.NewMasterTracker(allIDString, func(id string, v bool) {
 			val := 0.0
 			if v {
 				val = 1.0
 			}
-			isMaster.Set(val, strconv.FormatInt(id, 10))
+			isMaster.Set(val, id)
 		})
 	}
 
 	// Synchronize the set of configured log IDs with those we are tracking mastership for.
 	for _, logID := range allIDs {
-		knownLogs.Set(1.0, strconv.FormatInt(logID, 10))
+		knownLogs.Set(1, idToString[logID])
```

Comment:
Maybe you could avoid having `idToString` map by iterating over `allIDString` instead?

Reply:
later in the function, we use `logID` as an index into `electionRunner`.  In this function, both string and integral formats are needed. It's a bit awkward, but this is the only function that translates between `int64` and `string`

Reply:
We could make `electionRunner` map from `string` as well? The overhead seems to be equivalent to (even slightly less than) re-creating and accessing a string-to-int map on the flight.

Reply:
done